API_Guidelines.md

```diff
@@ -70,7 +70,37 @@ Payload
 ```
 
 ### GET multiple records using a query
-**Need documentation**. In the mean time [Finding Records](https://guides.emberjs.com/v2.10.0/models/finding-records/)
+Calling store.query() will make a `GET` request with the passed object serialized as query params. This method returns a DS.PromiseArray in the same way as findAll.
+
+Further reference: [Finding Records](https://guides.emberjs.com/v2.10.0/models/finding-records/)
```

Comment:
I would remove this whole section because (1) you're quoting the docs, so this should really be a quote, (2) you're linking to a specific version of the docs which will get outdated, (3) this should really be more of a recipe/guideline for apis, not documentation about the ember api itself